apps/admin/components/Auth/Poller.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component, Fragment } from 'react'
+import { Component, Fragment } from 'react';
```

Comment:
The code mod added semicolons – please re-run it on the current main on Wednesday without semicolons or remove them afterwards by running it all through prettier.